src/treelearner/feature_histogram.hpp

```diff
@@ -944,16 +944,20 @@ class FeatureHistogram {
         is_splittable_ = true;
         // better split point
         if (current_gain > best_gain) {
+        if (USE_MC) {
+          best_right_constraints = constraints->RightToBasicConstraint();
+          best_left_constraints = constraints->LeftToBasicConstraint();
+          if (best_right_constraints.min > best_right_constraints.max ||
+              best_left_constraints.min > best_left_constraints.max) {
+            continue;
+          }
+        }
```

Comment:
could fix the indent?

Reply:
I am sorry I am no sure what you mean. Is the indent here not the same way it is in the other "if" in the code, e.g. line 340?

Reply:
@CharlesAuguste I guess @guolinke meant the following indent:

![image](https://user-images.githubusercontent.com/25141164/103565287-26bd3e80-4ed1-11eb-8719-0dc0f103b7bc.png)